packages/backend/src/apps/toolbox/__tests__/common/condition-is-true.test.ts

```diff
@@ -94,6 +94,41 @@ describe('Condition is true', () => {
     expect(result).toEqual(expectedResult)
   })
 
+  /**
+   * Typical datetime formats:
+   * yyyy-MM-dd HH:mm
+   * dd MMM yyyy
+   * dd/MM/yyyy HH:mm:ss
+   * ISO format
+   */
+  it.each([
+    { text: '2024-11-06 12:00', expectedResult: true },
+    { text: '03/11/2024 12:00:30', expectedResult: false },
```

Comment:
i think new Date will parse this as mm/dd/yyyy instead of the intended dd/mm/yyyy. you can change this 05/11/2024 and expectedResult to true and test

Reply:
perhaps the only is to write a helper function that enumerates the common date formats until one fits.

Reply:
@pregnantboy ooo nice catch, but are you referring to doing this for test cases or doing that for the actual functionality?